ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/ProxyIntegrationWithPathTest.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.jsonrpcproxy;
+
+import java.io.IOException;
+
+import org.web3j.crypto.CipherException;
+
+class ProxyIntegrationWithPathTest extends ProxyIntegrationTest {
```

Comment:
This looks to be the same test as ProxyIntegrationWithPathAndReplaceTest?

Reply:
The derived tests have been removed, and in fact, the parameterisation has been removed - ultimately,  a blank string is basically the same as a string with content.